pyproject.toml

```diff
@@ -14,6 +14,7 @@ tiktoken = ">=0.4.0"
 importlib-metadata = ">=6.8.0"
 tokenizers = "*"
 click = "*"
+boto3 = ">=1.28.57"
```

Comment:
can we not add boto3 as a dependancy to litellm

Reply:
Yes, for that I struggle too, I've moved it out of dependency. Thanks.